src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -80,13 +80,20 @@
     <Compile Include="$(PathToMsalSources)\**\*.cs" Exclude="$(PathToMsalSources)\obj\**\*.*" />
     <Compile Remove="$(PathToMsalSources)\Platforms\**\*.*;$(PathToMsalSources)\Resources\*.cs" />
     <Compile Remove="$(PathToMsalSources)\PlatformsCommon\PlatformNotSupported\ApiConfig\SystemWebViewOptions.cs" />
+    <!-- Remove custom CertificateRequest from all targets, then conditionally include for downlevel frameworks only -->
+    <Compile Remove="$(PathToMsalSources)\ManagedIdentity\V2\CertificateRequest.cs" />
     <EmbeddedResource Include="$(PathToMsalSources)\Properties\Microsoft.Identity.Client.rd.xml" />
     <None Include="$(PathToMsalSources)\..\..\..\README.md" Pack="true" PackagePath="\" />
     <None Include="Platforms\net\JsonObjectAttribute.cs" />
     <None Include="Platforms\net\JsonStringConverter.cs" />
     <None Include="Platforms\net\MsalJsonSerializerContext.cs" />
   </ItemGroup>
 
+  <!-- Include custom CertificateRequest polyfill only for downlevel frameworks that don't have OtherRequestAttributes -->
+  <ItemGroup Condition="'$(TargetFramework)' != '$(TargetFrameworkNet)'">
```

Comment:
This csproj file is messy enough. I think I'd prefer to have #if in the code instead.

Reply:
done